x/merkledb/trie_test.go

```diff
@@ -1178,3 +1180,109 @@ func TestTrieViewInvalidChildrenExcept(t *testing.T) {
 	require.True(view3.invalidated)
 	require.Empty(view1.childViews)
 }
+
+
+func Test_Trie_ConcurrentReadWrite(t *testing.T) {
+	trie, err := getBasicDB()
+	require.NoError(t, err)
```

Comment:
nit: adding `require := require.New(t)` at the top of the test lets you drop the `t` in all these calls (and keeps it in-line with our other tests).
```suggestion
	require.NoError(err)
```